src/c++/perf_analyzer/request_rate_worker.h

```diff
@@ -122,6 +123,7 @@ class RequestRateWorker : public LoadWorker, public IScheduler {
 
   void CreateContextFinalize(std::shared_ptr<InferContext> ctx) override
   {
+    // FIXME: need to update this from max_threads to size of threads
```

Comment:
I think this story is the correct one to fix this case. The fix isn't trivial though. I think we need to change the constructor of RRW  -- the 'max_threads' variable should be renamed 'num_threads' or 'active_threads' --  and that correct value needs to be passed in to the constructor from RRM.

Reply:
updated